### PR TITLE
Fix white-space issues (#188)

### PR DESCRIPTION
**Pull request by UtiluMark**

### CODE REVIEW

File: extension/chrome/content/songbird.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ init: function()
 {
   // var brandbundle = document.getElementById("bundle_brand");
   var bundleSvc = Components.classes["@mozilla.org/intl/stringbundle;1"]
-  					.getService(Components.interfaces.nsIStringBundleService);
+                    .getService(Components.interfaces.nsIStringBundleService);
```
Comment (whimboo) [inline extension/chrome/content/songbird.js]: Lets align the leading dot with the `C` from Components. Similar below for the call to createBundle().

File: extension/chrome/skin/options/options.css [hunk 2]
```diff
@@ -1,40 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (whimboo) [inline extension/chrome/skin/options/options.css]: Those files have already been removed in another PR. So you might want to rebase against current master.

**Comment by whimboo** [comment]
@UtiluMark this PR looks fine to me. Thank you for all that white-space cleanup. I think we pushed this out for quite a while.  There are two small issues which would need to be fixed. Please let me know when you pushed another commit to fix that. I will merge the PR then. Thanks!

**Comment by whimboo** [comment]
Btw. this PR is another step to get issue #188 fixed.

**Comment by UtiluMark** [comment]
@whimboo Thanks. I've merged and fixed the alignment issue.

**Comment by whimboo** [comment]
Looks fine now. I will squash the commits and land on master in a moment. Thanks for that great clean-up. The editor will be happy now.

**Comment by whimboo** [comment]
Merged PR as:
https://github.com/mozilla/nightlytt/commit/dabd430df46a654fc11859d624ff640d08d392d6

**Pull request closed without merging**